### PR TITLE
logged out redirect

### PR DESCRIPTION
**Pull request by anuejn**
this can be used to redirect to about.transcribee.net

### CODE REVIEW

File: backend/transcribee_backend/config.py [hunk 1]
```diff
@@ -69,7 +71,10 @@ def get_short_page_config() -> Dict[str, ShortPageConfig]:
 
 
 def get_public_config():
-    return PublicConfig(models=get_model_config())
+    return PublicConfig(
+        models=get_model_config(),
+        logged_out_redirect_url=environ.get("TRANSCRIBEE_LOGGED_OUT_REDIRECT_URL"),
```
Comment (pajowu) [inline backend/transcribee_backend/config.py:76]: Please pull this from the `Setttings` instance, which gives more flexible ways of setting this (for example via a .env file)

File: frontend/src/app.tsx [hunk 2]
```diff
@@ -47,6 +48,26 @@ export function LoggedInRoute<T extends DefaultParams = DefaultParams>({
...
+}: RouteProps<T>) {
+  const [_, navigate] = useLocation();
+  const { isLoggedIn, isLoading } = useAuthData();
+  const { data: config, isLoading: configLoading } = useGetConfig({});
+  if (isLoading) {
+    return <Route component={LoadingPage} />;
+  }
+  if (!isLoggedIn && !configLoading) {
+    if (config.logged_out_redirect_url) {
+      window.location.replace(config.logged_out_redirect_url);
```
Comment (pajowu) [inline frontend/src/app.tsx:62]: Souldnt this be assign, so the user can go back?

**Review by pajowu — CHANGES_REQUESTED** [review]

File: frontend/src/app.tsx (hunk 2 above)
Comment (anuejn) [reply frontend/src/app.tsx:62]: If we use assign and the user gos back, they are landing in an infinite loop. That would be way worse ux imo

File: backend/transcribee_backend/config.py (hunk 1 above)
Comment (anuejn) [reply backend/transcribee_backend/config.py:76]: Aaah I did not understand that, sounds good :)

**Review by pajowu — APPROVED** [review]

**Pull request merged**